autoload/mysqlQuery.vim

```diff
@@ -0,0 +1,70 @@
+function! mysqlQuery#GetDescribeTableQuery(database, table)
+    let l:query = 'DESCRIBE ' . a:database . '.' . a:table
+    return l:query
```

Comment:
Can we leave all of this file in the `mysql` file?
I'm trying to keep the codebase small and feature-light so having it all in one file will highlight if it's getting too unwieldy